CMakeLists.txt

```diff
@@ -33,7 +33,7 @@ endif()
 include(GNUInstallDirs)
 
 add_subdirectory(lib)
-if(NOT BUILD_EXPORTED_TARGETS_ONLY)
+if(NOT BUILD_EXPORTED_TARGETS_ONLY AND BUILD_TESTING)
```

Comment:
This only works if you previously used `include(CTest)`

Reply:
Empirically, 9f812ae seemed to work correctly with cmake  3.18.4 as BUILD_TESTING was not defined.

Reply:
When reviewing https://github.com/fortran-lang/stdlib/pull/629 not including `CTest` did lead to problems, which were related to caching and restarting builds.